config/environments/production.rb

```diff
@@ -76,12 +76,12 @@
     address: Rails.application.secrets.smtp_address,
     port: Rails.application.secrets.smtp_port,
     authentication: Rails.application.secrets.smtp_authentication,
-    user_name: Rails.application.secrets.smtp_authentication_username,
+    user_name: Rails.application.secrets.smtp_username,
```

Comment:
Be careful!!!! that is needed to email works!

Reply:
It's ok, see `secrets.yml`L171

Reply:
Here `smtp_username` and `smtp_authentication_username` are different values.